versions.cmake

```diff
@@ -112,11 +112,11 @@ add_revision(lapack
   # these are the linux + osx binaries from PYPI downloaded and uploaded to our dependencies site.
   # We can download and install them from there
   add_revision(numpy
-    URL "https://www.tomviz.org/files/numpy-1.14.0-python3.6.tar.gz"
+    URL "https://www.tomviz.org/files/numpy-1.16.3-python3.7.tar.gz"
     URL_HASH "SHA512=41d8bd3e83072f2040af5d995e3700aaec431f021f56c40a364f3679baa38ca3c6e66dd51749d116461ffc425c8e1f0c5c2050b3f2e1760d60d7c74744e60db2")
 
   add_revision(scipy
-    URL "https://www.tomviz.org/files/scipy-1.0.0-python3.6.tar.gz"
+    URL "https://www.tomviz.org/files/scipy-1.2.1-python3.7.tar.gz"
```

Comment:
How are you not updating the hashes without this breaking? It would be good to take the opportunity to `s|https://www.tomviz.org/files/|${tv_dl_prefix}|` too so that we can simply modify the prefix if necessary. I guess all of these would need to be put in place.

Reply:
I was trying them out locally first to make sure the ones I have work (so that we wouldn't have to re-upload files and change the hashsums a second time if they didn't). I was trying them out in the Linux docker image, which is where I ran into issues with pyFFTW. But other than pyFFTW, the Linux binaries seemed to work okay.

Sure, we can add a prefix! Just let me know what it is so we can add it here.